test/e2e/storage/testsuites/multivolume.go

```diff
@@ -482,11 +482,11 @@ func TestConcurrentAccessToSingleVolume(f *framework.Framework, cs clientset.Int
 		ginkgo.By(fmt.Sprintf("Creating pod%d with a volume on %+v", index, node))
 		podConfig := e2epod.Config{
 			NS:            ns,
-			ImageID:       imageutils.DebianIptables,
 			PVCs:          []*v1.PersistentVolumeClaim{pvc},
-			SeLinuxLabel:  e2epv.SELinuxLabel,
+			SeLinuxLabel:  e2evolume.GetLinuxLabel(),
 			NodeSelection: node,
 			PVCsReadOnly:  readOnly,
+			ImageID:       e2evolume.GetTestImageId(imageutils.DebianIptables),
```

Comment:
why does this need to use iptables image?

Reply:
https://github.com/kubernetes/kubernetes/commit/f2bf2ab76ecfb43519e64dea2eef6051a5abaf31

Reply:
I think dd command has some special options that busybox cannot support